src/NzbDrone.Api/Music/ArtistResource.cs

```diff
@@ -26,23 +27,15 @@ public ArtistResource()
         public string AMId { get; set; }
```

Comment:
AllMusicID, I can change this to spell it out, it may be easier for future reference